### DISCUSSION TITLE
Proper retargetting animations

### DISCUSSION THREAD

**Discussion by woodencase01** [Sixze/ALS-Refactored#249] [General]
I'm not satisfied with the retarget pose from mesh, as my character has different proportions, and some legs and hands animations aren't looking good, especially when ragdolling. Also, the whole layered skeleton hierarchy makes it hard to work with in the sequencer. My goal would be to regarget everything and stop using a retarget pose from mesh.

I gave a try to retarget all animation, animation blueprint, montages and bring it back together. However, many things broke:
- The hands seems to be constantly in IK trying to snap at the character's back
- The mantling animations aren't animating properly

Is there a proper way to retarget the animations? There isn't much documenation and tutorials, sadly.

**Comment by Sixze** [comment]
I haven't really delved into retargeting and I'm not going to anytime soon, so I can only recommend where to start looking for a solution to your issue:

https://github.com/Sixze/ALS-Refactored/discussions/15
https://www.youtube.com/@tuatec/playlists
https://github.com/tuatec/TTToolbox
